packages/ocp-index/ocp-index.0.1.0/opam

```diff
@@ -17,5 +17,5 @@ depends: [
   "cmdliner"
 ]
 depopts: ["curses"]
-ocaml-version: [ >= "4.00.0" & < "4.02" ]
+available: [ocaml-version >= "4.00.0" & ocaml-version < "4.02" ]
```

Comment:
@lefessan Can you add parentheses around the terms? opam 1.2.0 is a bit confused by that (I think), see https://travis-ci.org/ocaml/opam-repository/jobs/129653866#L880 This PR seems to be the first occurrence of that error in the logs.